content/applications/general/companies/digest_emails.rst

```diff
@@ -2,82 +2,152 @@
 
 Navigate to Digest Emails by going to :menuselection:`Settings --> General Settings --> Statistics`,
-then activate the **Digest Emails** feature and click on save.
+then activate the :guilabel:`Digest Emails` feature and click on :guilabel:`Save`.
 
 .. image:: digest_emails/digest-email-settings.png
    :align: center
    :alt: Digest Emails section inside General Settings.
 
-You can control a variety of settings for your Digest Emails, such as:
+A variety of settings can be configured for Digest Emails, such as:
 
-- which KPIs are shared in the Digest
-- how often Digest Emails are sent
-- who in your organization receives Digest Emails
-- creating custom Digest Email templates
-- adding additional KPIs (Studio required)
+- Which KPIs are shared in the digest emails
+- How often digest emails are sent
+- Who in the organization receives digest emails
+- Creating custom digest email templates
+- Adding additional KPIs (Studio required)
 
 .. note::
    By default, Digest Email is *enabled*, and *Your Odoo Periodic Digest* serves as the primary
-   template, which includes all KPI measurements across your Odoo database and is sent daily to
+   template, which includes all KPI measurements across the Odoo database and is sent daily to
    administrators.
 
+.. warning::
+   When creating duplicates of databases that have sending capabilities (not testing-mode) the
+   digest emails will continue to send from the duplicate database unless deactivated. To deactivate
+   navigate to :menuselection:`Settings --> General Settings --> Statistics`. Then deactivate the
+   :guilabel:`Digest Emails` feature by un-ticking the checkbox and click on :guilabel:`Save`.
+
 .. _digest-emails/customize-digest:
 
 Customize *Your Odoo Periodic Digest*
 =====================================
 
 To customize the default Digest Email (*Your Odoo Periodic Digest*), go to :menuselection:`Settings
 --> General Settings --> Statistics --> Digest Email`, select *Your Odoo Periodic Digest* and click
-on the *external link* next to the dropdown selection.
+on the :guilabel:`external link` icon next to the drop-down menu selection.
 
 A popup window appears and presents a variety of editable settings, which include:
 
-- **Digest Title** - what you want your Digest Email to be called
-- **Periodicity** - control the regimen in how often Digest Emails are sent
-- **KPIs** - check/uncheck each calculated KPI that appears in Digest Emails
-- **Recipients** - add/remove users who receive your Digest Emails
-- **Custom** - add your own KPIs (Studio required)
+- :guilabel:`Digest Name` - the name of the digest email.
+- :guilabel:`Periodicity` - control the regimen in how often digest emails are sent.
+- :guilabel:`Next Send Date` - the date which the digest email will be sent again.
+- :guilabel:`KPIs` tab - check/uncheck each calculated KPI that appears in digest emails.
+- :guilabel:`Recipients` tab - add/remove users who receive the digest emails.
```

Comment:
```suggestion
- :guilabel:`Digest Name`: the name of the digest email.
- :guilabel:`Periodicity`: control how often digest emails are sent.
- :guilabel:`Next Send Date`: the date on which the digest email will be sent again.
- :guilabel:`KPIs` tab: check/uncheck each calculated KPI that appears in digest emails.
- :guilabel:`Recipients` tab: add/remove users who receive the digest emails.
```
I changed each dash to a colon, just because that's how I see lists written throughout documentation